manifests/src/upload_index.sh

```diff
@@ -29,23 +68,14 @@ else # Update existing dataset
 
     echo "Checking for changes in file contents: Remote ${REMOTE_HASH} vs Local ${LOCAL_HASH}"
     if [ "${REMOTE_HASH}" == "${LOCAL_HASH}" ]; then
-	echo "The urls and md5sums have not changed"
-	exit 0
+    	echo "The urls and md5sums have not changed"
```

Comment:
(not particularly important, but something funky happened with the spacing)

Reply:
Fixed